### DISCUSSION TITLE
What are possible ways to fetch image thumbnails from Data hub API query?

### DISCUSSION THREAD

**Discussion by Deriu** [pimcore/pimcore#14837] [Q&A / Support]
Hi, I'm trying to get the Pimcore Advanced Certificate.

I've done all chapters, except only one question, I'm trying 2 weeks and I'm confused :)

The question is about to fetch the thumbs with GraphQI:
"What are possible ways to fetch image thumbnails from Data hub API query?"

The four answers are:

Using Transformers for image attribute in the schema definition.
1) By providing thumbnail parameter on image attribute fullpath in the query.
2) By using GraphQL inline fragment.
3) By just requesting fullpath of image attribute.
4) The transformes I think is not mandatory. The third maybe is correct, because I need to write a queries as inline fragment.

After in video example if I write only fullpath I can't have thumb, so I've excluded it.

But if I put as correct 2 and 3 are wrong... If I add 4 as still wrong.

I'm confused..

**Comment by ali-mhanna** [comment]
![Screenshot 2023-12-12 115533](https://github.com/pimcore/pimcore/assets/152969110/529cebd3-3bfe-4310-a029-b8b1193aafc3)
